app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+    
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, class: "sing-input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autocomplete: "email", class: "sing-input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :photo  %><br />
+    <%= f.text_field :photo, class: "sing-input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio, class: "sing-input" %>
+  </div>
+  
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password", class: "sing-input" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password", class: "sing-input" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" , class: "sing-up-btn"%>
```

Comment:
- **[OPTIONAL]** Kindly refactor the form UIs to match the wireframe, where the elements are centered.